custom_components/ha_kia_hyundai/sensor.py

```diff
@@ -122,6 +128,30 @@ class KiaSensorEntityDescription(SensorEntityDescription):
     ),
 )
 
+SEAT_SENSOR_DESCRIPTIONS: Final[tuple[KiaSensorEntityDescription, ...]] = (
+    KiaSensorEntityDescription(
+        key="heated_driver_seat",
+        name="Seat-Driver",
+        exists_fn=lambda seat: bool(seat.front_seat_options[HEAT_VENT_TYPE]),
```

Comment:
this is where it throws an exception first, because front_seat_options returns an empty dict so key not found error I think (sorry my car is in the shop and I can't access the api for another week or two so I'm trying to just walk through it in my head :)